From bbe5aa1ec265b60edeeb6e746e3b8269db272cf7 Mon Sep 17 00:00:00 2001
From: Luca Weiss <luca@z3ntu.xyz>
Date: Fri, 20 Mar 2020 21:03:39 +0100
Subject: [PATCH 1/4] Add include to math.h

Fixes the following error:

colorutils.cpp: In static member function 'static qreal ColorUtils::contrast(const QColor&)':
colorutils.cpp:46:20: error: 'pow' was not declared in this scope
   46 |         redLight = pow((color.redF() + 0.055) / 1.055, 2.4);
      |                    ^~~
colorutils.cpp:52:22: error: 'pow' was not declared in this scope
   52 |         greenLight = pow((color.greenF() + 0.055) / 1.055, 2.4);
      |                      ^~~
colorutils.cpp:58:21: error: 'pow' was not declared in this scope
   58 |         blueLight = pow((color.blueF() + 0.055) / 1.055, 2.4);
      |                     ^~~
---
 src/UbuntuToolkit/colorutils.cpp | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/UbuntuToolkit/colorutils.cpp b/src/UbuntuToolkit/colorutils.cpp
index 602b164ec..e710593c4 100644
--- a/src/UbuntuToolkit/colorutils.cpp
+++ b/src/UbuntuToolkit/colorutils.cpp
@@ -18,6 +18,8 @@
 
 #include <QtGui/QColor>
 
+#include <math.h>
+
 UT_NAMESPACE_BEGIN
 
 ColorUtils::ColorUtils(QObject *parent) : QObject(parent)
-- 
2.25.2

